sw/device/silicon_creator/lib/sigverify/sigverify_tests/sigverify_set_testvectors.py

```diff
@@ -110,20 +109,14 @@ def main() -> int:
                         metavar='FILE',
                         type=argparse.FileType('r'),
                         help='Read test vectors from this HJSON file.')
-    tpl_default = open(
-        os.path.join(os.path.dirname(__file__), DEFAULT_TEMPLATE), 'r')
     parser.add_argument('--template',
                         metavar='FILE',
                         required=False,
```

Comment:
nit: I'm OK with removing the default file targets, but I think in this case the template file argument should be mandatory:
```suggestion
                        required=True,
```

Reply:
Ah right. the lines opening the default template file prior to the argument made bazel test failed. Anyhow I will merge this and change the lines.